tdrs-backend/Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN apt-get -y update
 RUN apt-get -y upgrade
 # Install a new package:
 RUN apt-get install -y gcc && apt-get install -y graphviz && apt-get install -y graphviz-dev
+RUN apt-get install -y libpq-dev python3-dev
```

Comment:
since you're not running docker anymore, do you need to add these libs?

Reply:
i'm not entirely sure why but, yes, the build fails without them. it may be the different postgres lib